runelite-client/src/main/resources/net/runelite/client/plugins/skillcalculator/skill_construction.json

```diff
@@ -60,6 +60,12 @@
       "name": "Crude Wooden Chair",
       "xp": 58
     },
+    {
+      "level": 1,
```

Comment:
According to the [wiki](https://oldschool.runescape.wiki/w/Long_bone) you need at least level 30 construction to be granted any xp.

Reply:
Ah I misread the page